### PR TITLE
feat: update header menu CCR-479

### PR DESCRIPTION
**Pull request by virtru-dubich**
### Proposed Changes

- @CCR-479
  - Created footer, header, main layout, custom icons (logo, avatar, arrow), dropdown with or styles for header menu

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [x] I have verified that my changes have not introduced new lint errors
- [ ] I have added a GitHub labels to the PR for either _patch_, `minor`, or **major**

### CODE REVIEW

File: lib/components/dropdown/topMenuDropdown.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  line-height: var(--vds-line-height-sm);
+}
+
+.avatarIcon {
+  margin-right: var(--vds-spacer-sm);
+}
+
+.arrow {
+  position: relative;
+  top: 1px;
```
Comment (ivanovSPvirtru) [inline lib/components/dropdown/topMenuDropdown.css:16]: If its relative it shouldn't work

File: lib/components/icon/customIconAvatar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { ReactComponent as UserAvatar } from '../../assets/user_avatar.svg';
+
+const CustomIconAvatar = ({ className }) => (
+  <UserAvatar className={cn('custom-avatar', { [className]: className })} />
```
Comment (ivanovSPvirtru) [inline lib/components/icon/customIconAvatar.js:7]: classNames('foo', 'bar'); // => 'foo bar' not sure that object needed here

File: lib/components/dropdown/topMenuDropdown.css (hunk 1 above)
Comment (virtru-dubich) [reply lib/components/dropdown/topMenuDropdown.css:16]: why? top works with relative

File: lib/components/layout/footer.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { Layout, Typography } from 'antd';
+import styles from './footer.css';
+
+const { Link } = Typography;
+const { Footer: FooterAntd } = Layout;
+
+const year = new Date().getFullYear();
+
+const Footer = () => (
+  <>
```
Comment (ivanovSPvirtru) [inline lib/components/layout/footer.js:11]: no need for this wrapper

File: lib/components/icon/customIconAvatar.js (hunk 2 above)
Comment (virtru-dubich) [reply lib/components/icon/customIconAvatar.js:7]: oh yes. that makes sense.

**Review by ivanovSPvirtru — APPROVED** [review]
lgtm

**Review by ivanovSPvirtru — APPROVED** [review]
lgtm

**Review by ivanovSPvirtru — APPROVED** [review]

**Pull request merged**